poser/morph_rotate_combine_poser.py

```diff
@@ -44,23 +45,23 @@ def pose_parameters(self) -> List[PoseParameter]:
     def get_morph_module(self):
         if self.morph_module is None:
             G = self.morph_module_spec.get_module().to(self.device)
-            G.load_state_dict(torch_load(self.morph_module_file_name))
+            G.load_state_dict(torch_load(self.morph_module_file_name, map_location=self.map_location))
```

Comment:
Have you tried just "map_location=self.device"? I'd prefer this one as it would be a lot simpler.

Reply:
Thanks for suggestion, it works pretty well.